pkgs/servers/web-apps/netbird-dashboard/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "netbird-dashboard";
```

Comment:
How close is the dashboard the original package? Should it be a passthru derivation of `pkgs.netbird` ?

i.e. `pkgs.netbird.dashboard` ?